src/features/dependencies/components/Dependencies.tsx

```diff
@@ -115,19 +64,19 @@ export const Dependencies = ({ dependencies, mode }: IDependenciesProps) => {
             </Box>
           }
           dataLength={listLength}
-          next={addDependencies}
+          next={next}
           scrollableTarget="infScroll"
           style={{ overflow: "hidden" }}
         >
-          {list.map((dependency, index) => (
+          {dependencies.map((dependency, index) => (
             <Box
               key={dependency.id}
               sx={{ marginBottom: index === listLength - 1 ? "0px" : "20px" }}
             >
               <DependenciesItem
                 mode={mode}
                 dependency={dependency}
-                onClick={handlePromote}
+                onClick={() => null}
```

Comment:
This event will be done in the edit version of this component. I don't want to do that here since this is more focused on read-only mode and would start to make this PR bloated.